lib/utils.pm

```diff
@@ -1932,4 +1933,15 @@ sub assert_secureboot_status {
     die "Error: SecureBoot is $statestr" if $state xor $expected;
 }
 
+sub upload_supportconfig_log {
+    my (%args) = @_;
+    $args{file_name} //= 'supportconfig.' . script_output("date '+%Y%m%d%H%M%S'");
+    $args{options}   //= '';
+    $args{timeout}   //= 600;
+    script_run("supportconfig -t . -B $args{file_name}", $args{timeout});
+    assert_script_run("tar zcvfP scc_$args{file_name}.tar.gz scc_$args{file_name}/");
```

Comment:
we used *supportconfig.xxx instead of 'scc_supportconfig.xxx' here, because we found for online installation, the directory to store supportconfig files is named scc_supportconfig.xxx, while for offline installation(with Full media), the directory is named nts_supportconfig.xxx. Our test should support both online and offline installation, so we suggest change here.

Reply:
@Julie-CAO thank you for very helpful comment, I actually also found also [bsc#](https://bugzilla.suse.com/show_bug.cgi?id=1166774) and I updated the code accordingly.

Reply:
I finally figured out the origin of 'nts' ... thanks! All the changes LGTM.